policyengine_us/parameters/gov/states/co/tax/income/credits/family_affordability/reduction/start.yaml

```
@@ -2,19 +2,24 @@ description: Colorado reduces the family affordability tax credit starting at th
 metadata:
   period: year
   unit: currency-USD
+  uprating:
+    parameter: gov.irs.uprating
+    rounding:
+      type: downwards
```

See:
>For income tax years commencing on or after January 1, 2025, the department of revenue is required to adjust the federal adjusted gross income amounts set forth in the bill to reflect inflation for each income tax year in which the credit is allowed if cumulative inflation since the last adjustment, when applied to the current limits, results in an increase of at least one thousand dollars when the adjusted limits are rounded to the nearest one thousand dollars.

do the same in increment.yaml

```suggestion
      type: nearest
```

---

looks like you resolved these without making the changes